nin/backend/compile.js

```diff
@@ -32,18 +33,44 @@ function renderError() {
     chalk.grey('[') + chalk.red('❌ ERROR') + chalk.grey(']'));
 }
 
-function createArchive() {
-  process.stdout.write(chalk.yellow('\nCreating zip archive'));
-  const outputStream = fs.createWriteStream('bin/demo.zip');
-  const archive = archiver('zip');
-  archive.pipe(outputStream);
-  archive.file('screenshot.png');
-  archive.glob('*.nfo');
-  archive.file('bin/demo.png.html', {
-    name: 'demo.png.html',
+function slugifyTitle(title) {
+  return slugify(title).toLowerCase();
+}
+
+async function createArchive(projectPath) {
+  return new Promise(resolve => {
+    process.stdout.write(chalk.yellow('\nCreating zip archive'));
+    const metadata = utils.getProjectMetadata(projectPath);
+    const slug = slugifyTitle(metadata.projectSettings.title);
+    const outputStream = fs.createWriteStream('bin/' + slug + '.zip');
+    const archive = archiver('zip');
+    archive.pipe(outputStream);
+    let foundErrors = false;
+    archive.on('warning', e => {
```

Comment:
Either way is fine. The way it is done now is consistent with the rest of the error handling in this file, so I think it is best to leave it as-is for now.